owlbot.py

```diff
@@ -40,13 +41,14 @@
 # Add templated files
 # ----------------------------------------------------------------------------
 templated_files = common.py_library(
-python.py_samples(skip_readmes=True)
     samples=False,  # set to True only if there are samples
     microgenerator=True,
     cov_level=98,
 )
 s.move(templated_files, excludes=[".coveragerc"])  # microgenerator has a good .coveragerc file
 
+python.py_samples(skip_readmes=True)
+
```

Comment:
We need to be including `owlbot.py` in the `lint` and `blacken` sessions to avoid being able to merge this kind of breakage in the future.